docs/get-started/getting-started.md

```diff
@@ -39,6 +39,11 @@ will vary depending on your cluster manager. Here are some example configs:
   - `--conf spark.task.resource.gpu.amount=1`
 - Specify a GPU discovery script (required on YARN and K8S):
   - `--conf spark.executor.resource.gpu.discoveryScript=./getGpusResources.sh`
+- Explain why some operations of a query were not placed on a GPU or not:
+  - `--conf spark.rapids.sql.explain=ALL` will display whether each operation is placed on GPU.
+  - `--conf spark.rapids.sql.explain=NONE` will disable the log of `rapids.sql.explain`.
```

Comment:
switch the order of None and NOT_ON_GPU.. I think it would make sense to have them listed by how much we output.